gthockey-iOS/gthockey-iOS/CustomUIComponents/PillButton.swift

```diff
@@ -47,6 +47,7 @@ class PillButton: UIButton {
         self.init()
 
         setTitle(title, for: .normal)
+        originalButtonText = title
         self.backgroundColor = backgroundColor
```

Comment:
I know this was my mistake, but can we try to get rid of all of the `self` calls in the three lines here. I like your method better.